uint8 RGB1 # 0 : tắt | 1 : Error (red) | 2 : move simple (green faded) | 3 : move special (green blink) | 4 : Perform (LightBlue faded) | 5 : Wait (Yellow faded) | 6 : Move Stop (Purple blink) | other : off
uint8 RGB2 # led sau
uint8 sound_type # 0 : tắt | 1 : oke | 2 : Warning | 3 : Error